nil/internal/collate/bootstrap.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/NilFoundation/nil/nil/internal/db"
 	"github.com/NilFoundation/nil/nil/internal/network"
 	"github.com/NilFoundation/nil/nil/internal/types"
+	"github.com/libp2p/go-libp2p/core/peer"
```

Comment:
I'd suggest to wrap it somehow because we use libp2p directly only inside `internal/network/ module.